packages/azure-kusto-ingest/source/retry.ts

```diff
@@ -19,10 +19,16 @@ export class ExponentialRetry {
             throw new Error("Max retries exceeded");
         }
 
-        const base = this.sleepBaseSecs * Math.pow(2, this.currentAttempt);
+        this.currentAttempt++;
```

Comment:
Can you explain this change? Will this affect ManagedStreaming?

Reply:
Did not change any logic, just skipping the sleep function when reach to the retry limit.